src/components/general/input/index.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components';
+
+const StyledInput = styled.input`
+  border-radius: 20px;
+  border: ${({ theme }) => theme.colors.dark} solid 1px;
+  margin: ${({ theme }) => theme.sizes.spacing.sm};
+  padding: ${({ theme }) => theme.sizes.spacing.sm}
+    ${({ theme }) => theme.sizes.spacing.lg};
+  font-size: ${({ theme }) => theme.sizes.fonts.lg};
+  font-weight: 300;
+`;
+
+interface InputProps extends React.ComponentPropsWithoutRef<'input'> {
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
```

Comment:
I *think* you don't need to define this here because it's already defined on the input element and you're already forwarding those props

Reply:
Thanks Erica, I've changed the type of InputProps slightly as when I removed onChange I was getting the error, "An interface declaring no members is equivalent to its supertype."